curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.english.md

```diff
@@ -2,6 +2,7 @@
 id: 56533eb9ac21ba0edf2244bf
 title: Local Scope and Functions
 challengeType: 1
+videoUrl: 'https://scrimba.com/c/cd62NhM'
```

Comment:
I occasionally removed the comments and 'use strict' lines when I created the videos since they didn't impact the code. I figured it was better to simplify the code when demonstrating so it would be easier to understand.